src/high-level/include/iDynTree/KinDynComputations.h

```diff
@@ -318,6 +383,18 @@ class KinDynComputations {
                        const iDynTree::VectorDynSize &s_dot,
                        const iDynTree::Vector3& world_gravity);
 
+    /**
+     * Set the state for the robot (fixed base)
+     * Same as setRobotState, but with:
+     *  world_T_base      = iDynTree::Transform::Identity()
+     *  base_velocity     = iDynTree::Twist::Zero();
+     * @warning the Span objects should point an already existing memory. Memory allocation and resizing cannot be achieved with this kind of objects.
+     * @return true if all went well, false otherwise.
```

Comment:
on success?

Reply:
Here I tried to follow the already existing documentation. 
E.g. 
https://github.com/robotology/idyntree/blob/0d71b87ffbdd2953d9df34d0f61edf24c73d2ad5/src/high-level/include/iDynTree/KinDynComputations.h#L218

https://github.com/robotology/idyntree/blob/0d71b87ffbdd2953d9df34d0f61edf24c73d2ad5/src/high-level/include/iDynTree/KinDynComputations.h#L289

https://github.com/robotology/idyntree/blob/0d71b87ffbdd2953d9df34d0f61edf24c73d2ad5/src/high-level/include/iDynTree/KinDynComputations.h#L690

Reply:
@traversaro let me know if I should change the entire documentation as:
```
@return true in case of success, false otherwise.
```

Reply:
I agree in general that `in case of success` is cleaner, but I think we can do this change in a different PR. Here adding the message consistently with the rest of the class make sense.

Reply:
Ok we can open a new PR once this has been merged

Reply:
SG.